src/components/NcActionButton/NcActionButton.vue

```diff
@@ -567,7 +567,13 @@ export default {
 @include action--disabled;
 @include action-item('button');
 
-.action-button__pressed-icon {
-	margin-inline: auto calc($icon-margin * -1);
+.action-button {
+	&__pressed-icon {
+		margin-inline: auto calc($icon-margin * -1);
+	}
+
+	&__text {
```

Comment:
To make sure, it works for everything in the button with any layout/icons and etc.
```suggestion
	* {
```

Reply:
Good point~~er~~